LICENCE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2015-2017 Shiftgig Inc.
```

Comment:
I haven't seen that before, but I like the idea since it makes recognizing it *a lot* easier.